services/brig/test/integration/API/Provider.hs

```diff
@@ -105,17 +98,41 @@ tests conf p db b c g = do
             , test p "delete"                 $ testDeleteService conf db b
             ]
         , testGroup "bot"
-            [ test p "add-remove" $ testAddRemoveBot conf crt db b g c
-            , test p "message"    $ testMessageBot conf crt db b g c
+            [ test p "add-remove" $ testAddRemoveBot conf db b g c
+            , test p "message"    $ testMessageBot conf db b g c
+            , test p "bad fingerprint" $ testBadFingerprint conf db b g c
             ]
         , testGroup "bot-teams"
-            [ test p "add-remove"  $ testAddRemoveBotTeam conf crt db b g c
-            , test p "message"     $ testMessageBotTeam conf crt db b g c
-            , test p "delete conv" $ testDeleteConvBotTeam conf crt db b g c
-            , test p "delete team" $ testDeleteTeamBotTeam conf crt db b g c
+            [ test p "add-remove"  $ testAddRemoveBotTeam conf db b g c
+            , test p "message"     $ testMessageBotTeam conf db b g c
+            , test p "delete conv" $ testDeleteConvBotTeam conf db b g c
+            , test p "delete team" $ testDeleteTeamBotTeam conf db b g c
             ]
         ]
 
+----------------------------------------------------------------------------
+-- Config
+
+data Config = Config
+    { privateKey   :: FilePath
+    , publicKey    :: FilePath
+    , cert         :: FilePath
+    , botHost      :: Text
+    , botPort      :: Int
+    } deriving (Show, Generic)
+
+instance FromJSON Config
+
+-- | Get the config from environment variables (and some defaults)
```

Comment:
I moved all `getEnv`s into one place so that I would be able to rely on the `Config` being a `Just` in all places